formbricks/formbricks
# fix:changed the label on connection indicator

## What does this PR do?

Fixed the connection label indicator.

Fixes # (issue)



## Type of change




## How should this be tested?

1. Head to the connection section, where all your connected apps are shown
2. Now check, prerviously Webhooks section used to show "zaps", but not it will show "Webhooks" and Zapier will show "Zaps"

## Checklist



### Required


### Appreciated
